stages/org.osbuild.systemd.journald-storage

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python3
+"""
+Configures journald's storage options.
+It will set "persistent" as default and data will we stored on disk.
+
+See:
+https://bugzilla.redhat.com/show_bug.cgi?id=1265295
+https://pagure.io/workstation-ostree-config/blob/main/f/fedora-common-ostree.yaml#_103
+https://github.com/coreos/fedora-coreos-config/blob/testing-devel/overlay.d/05core/usr/lib/systemd/journald.conf.d/10-coreos-persistent.conf
+"""
+
+import sys
+import osbuild.api
+
+SCHEMA = """
+"additionalProperties": false,
+"properties": {
+  "storage_type": {
+    "type": "string",
+    "description": "Sets where to store journal data; on disk by default"
+  }
+}
+"""
+
+
+def main(tree, options):
+    storage_type = options.get("storage_type", "persistent")
+    with open(f"{tree}/usr/lib/systemd/journald.conf.d/10-persistent.conf",
+              encoding="utf8") as f:
+        conf = f"[Journal]\nStorage={storage_type}"
+        f.write(conf)
```

Comment:
and perhaps just error out if no option has been specified

Reply:
> and perhaps just error out if no option has been specified

We tend to prefer marking it as "required" to avoid this case, but yeah, the result is the same.

However, if you happen to go the `org.osbuild.systemd-journald`-route, this would probably not be a suitable approach, since you would force every caller to specify a storage option, even if they don't care.